InterviewQuestions/Chapters/BitInsertions/BitInsertion.cs

```diff
@@ -0,0 +1,20 @@
+namespace InterviewQuestions.Chapters.BitInsertion
+{
+    public class BitInsertions
+    {
+        public int BitInsertionSolution(int N,int M, int i, int j)
```

Comment:
Please give appropriate names for arguments. what are those i,j M, and N?
Are i and j startIndex and endIndex?

Reply:
arguments renamed, you can look here fb2ebc838d9c505ba096e3ef57dbb4cfa4f01e9a